Setup/UpgradeSchema.php

```diff
@@ -86,5 +86,51 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
 
             $installer->endSetup();
         }
+
+        if (version_compare($context->getVersion(), '1.0.3', '<')) {
+            $installer = $setup;
+            $installer->startSetup();
+
+            /**
+             * Create table 'tj_product_tax_categories'
+             */
+            $table = $installer->getConnection()
+                ->newTable($installer->getTable('tj_product_tax_categories'))
+                ->addColumn(
+                    'id',
+                    \Magento\Framework\DB\Ddl\Table::TYPE_INTEGER,
+                    null,
+                    ['identity' => true, 'unsigned' => true, 'nullable' => false, 'primary' => true],
+                    'Product Tax Code'
+                )->addColumn(
+                    'product_tax_code',
```

Comment:
@ryanreeves-taxjar Should we set up an index for `product_tax_code` since we're loading categories solely by that column in Observer/ImportCategories.php to prevent duplicate categories when importing?

Reply:
Good idea, added an index to that column.